shared/templates/service_disabled/ansible.template

```diff
@@ -11,9 +11,10 @@
         - name: Disable service {{{ SERVICENAME }}}
           systemd:
             name: "{{{ DAEMONNAME }}}.service"
-            enabled: "no"
+            enabled: false
             state: "stopped"
-            masked: "yes"
+            masked: true
+          when: ansible_facts.services["{{{ DAEMONNAME }}}.service"] is defined 
```

Comment:
After this conditional, two rules are failing in CI tests. Could you check and fix the errors reported by CI tests, please?